integration-test/skeleton/serverless.yml

```diff
@@ -33,4 +33,9 @@ custom:
       - one
       - two
       - --three
-    docker: DOCKER
+    # environment variables appear as strings. Index that string into a
+    # dictionary with actual boolean values to get the boolean back
+    docker: ${self:custom.boolean.${env:FORCE_DOCKER, "DOCKER_DEFAULT"}}
+  boolean:
```

Comment:
why is this necessary?

Reply:
Or do I really not want to know?

Reply:
There's the comment above: environment variables are strings, so if you say

```yaml
docker: ${env:FORCE_DOCKER}
```

it'll expand to:

```yaml
docker: "false"
```

However, you can use `self:custom` to get a value _out of your file_, and indirection is allowed, so `self:custom.boolean.true` expands to a _boolean_ value `true`.